pkg/kubelet/server/stats/summary.go

```diff
@@ -304,10 +301,12 @@ func (sb *summaryBuilder) containerInfoV2ToNetworkStats(info *cadvisorapiv2.Cont
 		txErrors += inter.TxErrors
 	}
 	return &NetworkStats{
-		RxBytes:  &rxBytes,
-		RxErrors: &rxErrors,
-		TxBytes:  &txBytes,
-		TxErrors: &txErrors,
+		Time:      unversioned.NewTime(cstat.Timestamp),
+		StartTime: unversioned.NewTime(info.Spec.CreationTime),
```

Comment:
For all the cumulative metrics, the StartTime of the container should be valid. Why replicate it?